container/create_image_config.py

```diff
@@ -76,10 +82,11 @@
 parser.add_argument('--null_cmd', action='store', default=False,
                     help='If True, "Cmd" will be set to null.')
 
-_PROCESSOR_ARCHITECTURE = 'amd64'
-
-_OPERATING_SYSTEM = 'linux'
+parser.add_argument('--architecture', action='store', default='amd64',
```

Comment:
It's not actually used right now.  Docker does support other architectures though so it may make sense going forward.  Would you like me to revert this part of the change?